<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddTextMultiPosDialog</class>
 <widget class="QDialog" name="AddTextMultiPosDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>363</width>
    <height>776</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Добавить текст для маркировки пломб</string>
  </property>
  <widget class="QToolButton" name="toolButtonFontPath">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>328</x>
     <y>379</y>
     <width>27</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxFontInternal">
   <property name="geometry">
    <rect>
     <x>82</x>
     <y>402</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>шрифт по умолчанию</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>674</y>
     <width>281</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>При заполнении выводить каждую n-ю линию:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>360</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Приращение:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>189</x>
     <y>527</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Смещение Y:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>698</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Способ рисования букв:</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinOffsetX">
   <property name="geometry">
    <rect>
     <x>97</x>
     <y>522</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>99999</number>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>333</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Начальный номер:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditFont">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>82</x>
     <y>379</y>
     <width>241</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboWayOfDrawing">
   <property name="geometry">
    <rect>
     <x>163</x>
     <y>694</y>
     <width>191</width>
     <height>22</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <item>
    <property name="text">
     <string>Заливка</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Контур</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Контур и заливка</string>
    </property>
   </item>
  </widget>
  <widget class="QSpinBox" name="spinOffsetY">
   <property name="geometry">
    <rect>
     <x>275</x>
     <y>522</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>99999</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinVertCount">
   <property name="geometry">
    <rect>
     <x>275</x>
     <y>550</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>99999</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>525</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Смещение X:</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinHorCount">
   <property name="geometry">
    <rect>
     <x>97</x>
     <y>550</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>99999</number>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>136</x>
     <y>736</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>542</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Количество
столбцов:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>381</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Шрифт:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>542</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Количество
строк:</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinStartNumber">
   <property name="geometry">
    <rect>
     <x>204</x>
     <y>329</y>
     <width>77</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>-999999999</number>
   </property>
   <property name="maximum">
    <number>999999999</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinStep">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>354</y>
     <width>226</width>
     <height>22</height>
    </rect>
   </property>
   <property name="suffix">
    <string/>
   </property>
   <property name="minimum">
    <number>-9999999</number>
   </property>
   <property name="maximum">
    <number>9999999</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>285</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Первая строка:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineFirstLine">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>280</y>
     <width>226</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>309</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Вторая строка:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineSecondLine">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>304</y>
     <width>226</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>570</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Интервал
вертикаль:</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinVertInterval">
   <property name="geometry">
    <rect>
     <x>275</x>
     <y>579</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>99999</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_17">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>570</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Интервал
горизонталь:</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinHorInterval">
   <property name="geometry">
    <rect>
     <x>97</x>
     <y>579</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>99999</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinPrintEveryLine">
   <property name="geometry">
    <rect>
     <x>297</x>
     <y>669</y>
     <width>58</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>999999</number>
   </property>
   <property name="value">
    <number>30</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>647</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Межстрочное расстояние:</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinLinesDistance">
   <property name="geometry">
    <rect>
     <x>175</x>
     <y>644</y>
     <width>181</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>0</number>
   </property>
   <property name="maximum">
    <number>9999</number>
   </property>
   <property name="value">
    <number>1000</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_30">
   <property name="geometry">
    <rect>
     <x>194</x>
     <y>427</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Масштаб X:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_28">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>427</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Размер строк:</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinScaleXNum">
   <property name="geometry">
    <rect>
     <x>275</x>
     <y>473</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <double>0.000000000000000</double>
   </property>
   <property name="maximum">
    <double>300.000000000000000</double>
   </property>
   <property name="value">
    <double>1.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_35">
   <property name="geometry">
    <rect>
     <x>194</x>
     <y>501</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Масштаб Y:</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinScaleYStr">
   <property name="geometry">
    <rect>
     <x>275</x>
     <y>449</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <double>0.000000000000000</double>
   </property>
   <property name="maximum">
    <double>300.000000000000000</double>
   </property>
   <property name="value">
    <double>1.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinSizeNum">
   <property name="geometry">
    <rect>
     <x>113</x>
     <y>473</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <double>300.000000000000000</double>
   </property>
   <property name="value">
    <double>8.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_34">
   <property name="geometry">
    <rect>
     <x>194</x>
     <y>475</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Масштаб X:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_33">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>475</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Размер номера:</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinSizeStr">
   <property name="geometry">
    <rect>
     <x>113</x>
     <y>425</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <double>300.000000000000000</double>
   </property>
   <property name="value">
    <double>8.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_32">
   <property name="geometry">
    <rect>
     <x>194</x>
     <y>452</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Масштаб Y:</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinScaleYNum">
   <property name="geometry">
    <rect>
     <x>275</x>
     <y>498</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <double>0.000000000000000</double>
   </property>
   <property name="maximum">
    <double>300.000000000000000</double>
   </property>
   <property name="value">
    <double>1.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinScaleXStr">
   <property name="geometry">
    <rect>
     <x>275</x>
     <y>425</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <double>0.000000000000000</double>
   </property>
   <property name="maximum">
    <double>300.000000000000000</double>
   </property>
   <property name="value">
    <double>1.000000000000000</double>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineStartNumber_prefix">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>329</y>
     <width>72</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineStartNumber_postfix">
   <property name="geometry">
    <rect>
     <x>283</x>
     <y>329</y>
     <width>72</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>623</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Радиус:</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinRadius">
   <property name="geometry">
    <rect>
     <x>245</x>
     <y>620</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>0</number>
   </property>
   <property name="maximum">
    <number>9999</number>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>603</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Дополнительное
межбуквенное
расстояние:</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinAdditionalLetterSpace">
   <property name="geometry">
    <rect>
     <x>97</x>
     <y>620</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>0</number>
   </property>
   <property name="maximum">
    <number>9999</number>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinTextMPPictureDrawEveryLine">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>86</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>99999</number>
   </property>
   <property name="value">
    <number>30</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_112">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>118</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Выводить картинку:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_113">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>143</y>
     <width>191</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Расстояние от картинки до текста:</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinTextMPPictureWidth">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>121</x>
     <y>59</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleTextMPPictureScaleY">
   <property name="geometry">
    <rect>
     <x>279</x>
     <y>33</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="value">
    <double>1.000000000000000</double>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonPictureLoad_2">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>6</y>
     <width>31</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_47">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>64</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Ширина:</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioTextMPPictureFixed">
   <property name="geometry">
    <rect>
     <x>19</x>
     <y>63</y>
     <width>16</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_104">
   <property name="geometry">
    <rect>
     <x>199</x>
     <y>39</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Масштаб Y:</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinTextMPTextDistance">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>138</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineTextMPBitmapPath">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>8</y>
     <width>201</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_103">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>82</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Выводить каждую n-ю линию
при заливке картинки:</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioTextMPPictureScale">
   <property name="geometry">
    <rect>
     <x>19</x>
     <y>39</y>
     <width>16</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="comboTextMPPicturePos">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>112</y>
     <width>201</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>сверху</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>справа</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>снизу</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>слева</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_101">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>0</y>
     <width>80</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Путь к файлу
картинки:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_105">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>39</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Масштаб X:</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleTextMPPictureScaleX">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>33</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="value">
    <double>1.000000000000000</double>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graphicsViewTextMPPicture">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>165</y>
     <width>121</width>
     <height>111</height>
    </rect>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinTextMPPictureHeight">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>59</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_102">
   <property name="geometry">
    <rect>
     <x>199</x>
     <y>64</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Высота:</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineTextMPBitmapPath</tabstop>
  <tabstop>buttonPictureLoad_2</tabstop>
  <tabstop>radioTextMPPictureScale</tabstop>
  <tabstop>doubleTextMPPictureScaleX</tabstop>
  <tabstop>doubleTextMPPictureScaleY</tabstop>
  <tabstop>radioTextMPPictureFixed</tabstop>
  <tabstop>spinTextMPPictureWidth</tabstop>
  <tabstop>spinTextMPPictureHeight</tabstop>
  <tabstop>spinTextMPPictureDrawEveryLine</tabstop>
  <tabstop>comboTextMPPicturePos</tabstop>
  <tabstop>spinTextMPTextDistance</tabstop>
  <tabstop>graphicsViewTextMPPicture</tabstop>
  <tabstop>lineFirstLine</tabstop>
  <tabstop>lineSecondLine</tabstop>
  <tabstop>lineStartNumber_prefix</tabstop>
  <tabstop>spinStartNumber</tabstop>
  <tabstop>lineStartNumber_postfix</tabstop>
  <tabstop>spinStep</tabstop>
  <tabstop>lineEditFont</tabstop>
  <tabstop>toolButtonFontPath</tabstop>
  <tabstop>checkBoxFontInternal</tabstop>
  <tabstop>doubleSpinSizeStr</tabstop>
  <tabstop>doubleSpinScaleXStr</tabstop>
  <tabstop>doubleSpinScaleYStr</tabstop>
  <tabstop>doubleSpinSizeNum</tabstop>
  <tabstop>doubleSpinScaleXNum</tabstop>
  <tabstop>doubleSpinScaleYNum</tabstop>
  <tabstop>spinOffsetX</tabstop>
  <tabstop>spinOffsetY</tabstop>
  <tabstop>spinHorCount</tabstop>
  <tabstop>spinVertCount</tabstop>
  <tabstop>spinHorInterval</tabstop>
  <tabstop>spinVertInterval</tabstop>
  <tabstop>spinAdditionalLetterSpace</tabstop>
  <tabstop>spinRadius</tabstop>
  <tabstop>spinLinesDistance</tabstop>
  <tabstop>spinPrintEveryLine</tabstop>
  <tabstop>comboWayOfDrawing</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
